var/spack/repos/builtin/packages/py-tensorflow-addons/package.py

```diff
@@ -33,7 +34,8 @@ class PyTensorflowAddons(Package):
     depends_on('py-pip', type='build')
 
     depends_on('py-setuptools', type='build')
-    depends_on('py-tensorflow@2.3:2.5', type=('run'))
+    depends_on('py-tensorflow@2.3:2.5', type=('run'), when='@0.13:')
```

Comment:
```suggestion
    depends_on('py-tensorflow@2.5', type=('run'), when='@0.13:')
```
I think it would be good to constrain this a bit more, since the binary addons are tied to the TF version.